docs/design/non-glibc-containers.md

```diff
@@ -0,0 +1,45 @@
+# Non-glibc Containers
+
+If you want to use a non-glibc-based container, such as Alpine Linux, you will need to arrange a few things on your own.
+First, you must supply your own copy of Node.js.
+Second, you must add a label to your image telling the agent where to find the Node.js binary.
+Finally, stock Alpine doesn't come with other dependencies that Azure Pipelines depends on:
+bash, sudo, which, and groupadd.
+
+## Bring your own Node.js
+You are responsible for adding a Node LTS binary to your container.
+As of November 2018, we expect that to be Node 10 LTS.
+You can start from the `node:10-alpine` image.
+
+## Tell the agent about Node.js
+The agent will read a container label "com.azure.dev.pipelines.handler.node.path".
+If it exists, it must be the path to the directory containing Node.js.
```

Comment:
@TingluoHuang I made an arbitrary choice to point to a directory, but maybe this should point directly to the executable?

Reply:
Changed this to point directly at the executable.